File: DSA/Asmi_Garg/day2_ques1.java
public class day2_ques1 {
    public static void main(String[] args) {
        int[] arrival_times = { -1, -3, 4, 2 };
        int threshold = 3;
        checkifcanceled(arrival_times, threshold);

    }

    static void checkifcanceled(int[] arr, int threshold) {
        int count = 0;
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] > 0) {
                count++;
            }
        }
        if (count >= threshold) {
            System.out.println("NO");
        } else {
            System.out.println("YES");

        }
    }
}

File: DSA/Shreyashi-Manik-Srivastava/Day2_Ques1.c
#include<stdio.h>

void cancel(int arr[], int n, int k){
    int count=0;
    for(int i=0;i<n;i++){
        if(arr[i]<0){
            count++;
        }
    }
    if(count>=k){
        printf("No");
    }
    else{
        printf("Yes");
    }
}
int main(){
    int n, k;
    printf("Enter size of the array: ");
    scanf("%d", &n);

    int arr[n];
    printf("Enter array elements:\n");
    for(int i=0; i<n; i++){
        scanf("%d", &arr[i]);
    }
    printf("Enter the threshold number:");
    scanf("%d",&k);

    cancel(arr,n,k);
}